src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -837,6 +807,25 @@ $row-data-header-width: 200px;
 }
 
 
+// Internal tags and type badges for matrix pages.
+.matrix-title-badge {
```

Comment:
You have used "matrix-title-badge " a number of time. It does not seem to follow the CSS naming standard we use. I forget the name but has a format like <object>_region__subregion.